$OpenBSD: patch-Makefile_PL,v 1.4 2012/04/07 20:54:07 sthen Exp $

Module::Install::AuthorTests is missing in ports tree,
but not really essential.

Same for Module::Install::Repository.


--- Makefile.PL.orig	Tue Jan 17 16:23:57 2012
+++ Makefile.PL	Sat Apr  7 21:53:04 2012
@@ -11,10 +11,9 @@ requires 'Term::ANSIColor'; # for youtube-download and
 requires 'XML::TreePP'; # for youtube-playlists
 
 tests 't/*.t t/*/*.t';
-author_tests 'xt';
 
 install_script 'bin/youtube-download', 'bin/youtube-playlists';
 
 test_requires 'Test::More', '0.96';
-auto_set_repository;
+#auto_set_repository;
 WriteAll;
